src/data/default_vocab.cpp

```diff
@@ -199,10 +199,6 @@ class DefaultVocab : public VocabBase {
             "Specified vocab directory {} does not exist",
             dir.string());
 
-    ABORT_IF(!dir.empty() && !filesystem::canWrite(dir),
-            "No write permission in vocab directory {}",
-            dir.string());
```

Comment:
As above, we need to remember that this might need updating one of the regression tests.

Reply:
I think our regression tests will remind us :)